c++/src/kj/async-test.c++

```diff
@@ -733,6 +746,45 @@ TEST(Async, TaskSet) {
   EXPECT_EQ(1u, errorHandler.exceptionCount);
 }
 
+TEST(Async, LargeTaskSetDestruction) {
```

Comment:
Minor point, but probably commits should be ordered so that the test for a bug is added _after_ the bug is fixed, otherwise there's a point in the commit history where the tests are broken, which may interfere with things like `git bisect`.

Reply:
Ok, I'll re-order. I usually prefer having the broken commit first so that you know that you really fixed the issue, but I can see the annoying impact it might have on git bisect.